airbyte-integrations/connectors/source-opsgenie/source_opsgenie/schemas/services.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
```

Comment:
@adamschmidt again more fields located on the API https://docs.opsgenie.com/docs/incident-api